File: src/model/Player.java
package model;

/**
 * Created by Nathanael on 01.11.2016.
 */
public class Player {
    private boolean alive;
    private String name;
    private Tile tile;

    public Player(String name, Tile start) {
        this.name = name;
        tile = start;
    }

    public void move(Tile tile) {

    }

    public void placeBomb() {

    }

    public String getName() {
        return name;
    }

    public Tile getTile() {
        return tile;
    }
}

File: src/model/Player.kt
package model

/**
 * storage class to store Players
 * Created by Nathanael on 01.11.2016.
 */
class Player(val name: String, start: Tile) {
    var isAlive: Boolean = false
    var tile: Tile? = null
        private set

    init {
        tile = start
    }

    fun move(tile: Tile) {
        this.tile = tile
    }

    fun placeBomb() {

    }
}
